tensorflow/compiler/xla/service/plaidml/tests/plaidml_logical_op_test.cc

```diff
@@ -0,0 +1,211 @@
+// Tests that show HLO Module conversion to PlaidML Program.
+
+#include <algorithm>
+#include <string>
+#include <map>
+#include <variant>
+
+#include <gtest/gtest.h>
+
+#include "absl/strings/str_cat.h"
+#include "tensorflow/compiler/xla/service/plaidml/compiler.h"
+#include "tensorflow/compiler/xla/service/plaidml/tests/plaidml_codegen_test.h"
+#include "tensorflow/compiler/xla/service/hlo_computation.h"
+#include "tensorflow/compiler/xla/service/hlo_evaluator.h"
+#include "tensorflow/compiler/xla/tests/filecheck.h"
+#include "tensorflow/compiler/xla/tests/test_utils.h"
+#include "tensorflow/core/lib/core/status_test_util.h"
+#include "tensorflow/core/platform/test.h"
+#include "plaidml/testenv.h"
+
+using ::plaidml::edsl::TensorBuffers;
+
+namespace xla {
+namespace plaidml {
+namespace {
+
+using TestCaseVal = std::vector<std::vector<int>>;
+using TestCasePairs = std::map<TestCaseVal, TestCaseVal>;
+
+struct EltwiseTestSpec {
```

Comment:
Do we want to be consistent with the file name and change references of `Eltwise` to `Logical`?

It is correct to call them eltwise, but logical would be more specific.